resources/sushy-tools/Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM registry.hub.docker.com/library/python:3.9
 
+ENV SUSHY_TOOLS_VERSION="0.18.1"
```

Comment:
for a followup, would be probably interesting to convert both ENVs to ARGs to give flexibility during build time

Reply:
If I [read correctly](https://docs.docker.com/engine/reference/builder/#arg), ARGs are used for builds only and ENVs for build+run. This one definitely is an ARG type as it is required in the build phase, but the other one is definitely a 'run' TMK.
WDYT?

Reply:
you can set an ENV to be the value of an ARG to give more flexibility during build time, I should have probably started with this :)

Reply:
> you can set an ENV to be the value of an ARG to give more flexibility during build time, I should have probably started with this :)

TIL, thanks!